src/types/metadata.ts

```diff
@@ -19,17 +19,17 @@ export interface VaultMetadata {
   totalSupply: Integer;
 
   apy?: Apy;
-  displayName: string;
+  displayName?: string | null;
```

Comment:
Not sure the reason to make this optional, maybe API not always has it, but we should aim to always fill it since frontend needs it.

Reply:
![telegram-cloud-document-5-6204086933451179407](https://user-images.githubusercontent.com/78794805/160544505-bdd82a58-8951-4cf4-9b07-a6ca4fa79329.jpg)

As discussed, the reason is the above, I'll have a look if this is used elsewhere. As you said, we might need to keep having this as optional if expected in the frontend to be present -- I'll check that

Reply:
Brought back the old interface with https://github.com/yearn/yearn-sdk/pull/263/commits/ce509c699c8ed1d18421af7ae93063b9701d6071

However that vault without `displayName` nor `migrationAvailable` should be fixed, it's the only one AFAIK that does not have those properties. The interface is not used anywhere else either.